drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -13,8 +13,9 @@ class JsonForms
 
     DATA_DIR = 'drivers/hmis/lib/form_data'
 
-    def initialize(env_key: nil)
+    def initialize(env_key: nil, override_generate_cdeds_in_test: false)
```

Comment:
`enable_cded_generation_in_test` might be clearer? "override" isn't really clear whether it's overriding it to generate or not generate